src/client/components/tasks.js

```diff
@@ -93,7 +94,7 @@ class TaskView extends DataComponent {
     let { document, bus } = this.props;
     let done = document.submitted() || document.published();
     let incompleteEles = this.props.document.elements().filter(ele => {
-      return !ele.completed() && !ele.isInteraction();
+      return !ele.completed() && !ele.isInteraction() && !ele.type() === ENTITY_TYPE.COMPLEX;
     });
 
     let ntfns = document.entities().filter(ele => !ele.completed()).map(ele => {
```

Comment:
This needs to filter out the `ele`s that are type `complex` as well.

Reply:
Isn't that just the same as `ENTITY_TYPE.COMPLEX`?

Reply:
According to this condition, any complex is always excluded from the incomplete element list.

Reply:
I'm referring to the `ntfns` - shouldn't this filter out complexes like in `incompleteEles`? Right now in the UI I still see the complexes as being listed as incomplete.

Reply:
<img width="859" alt="image" src="https://user-images.githubusercontent.com/4706307/76986281-d8c25480-6917-11ea-82a2-50a557363070.png">

Reply:
That issue is only reproducible with the extra node in the top-left of your screenshot.  With the change, this case look OK:

<img width="1232" alt="Screen Shot 2020-03-20 at 11 28 05 AM" src="https://user-images.githubusercontent.com/989043/77178789-f6b9c180-6a9d-11ea-8827-0de9f97764b9.png">